### PR TITLE
Delete code that is queued for next release

### PR DESCRIPTION
**Pull request by ValarDragon**
Delete this code which was to be removed in next release.

### CODE REVIEW

File: x/superfluid/keeper/epoch.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func (k Keeper) AfterEpochStartBeginBlock(ctx sdk.Context) {
 
 	ctx.Logger().Info("Distribute Superfluid gauges")
 	//nolint:errcheck
-	osmoutils.UnmeteredApplyFuncIfNoError(ctx, func(cacheCtx sdk.Context) error {
+	osmoutils.ApplyFuncIfNoError(ctx, func(cacheCtx sdk.Context) error {
```
Comment (czarcas7ic) [inline x/superfluid/keeper/epoch.go:36]: For this and the ones below, can you change ApplyFuncIfNoError to do the same as the one in hooks.go to print an error log so it's not failing completely silently?

**Review by czarcas7ic — APPROVED** [review]
If you agree with my comment I think we should include the error log prints, otherwise LGTM

File: x/superfluid/keeper/epoch.go (hunk 1 above)
Comment (ValarDragon) [reply x/superfluid/keeper/epoch.go:36]: i don't get it. ApplyFuncIfNoError does this for you
```
ctx.Logger().Error(err.Error())
```

Comment (ValarDragon) [reply x/superfluid/keeper/epoch.go:36]: Oh its double logged in hooks, wiht epoch as context. Imo this is fine, but if you'd like we can. This is only doing something for panics which contains the stack trace anyway.

Comment (czarcas7ic) [reply x/superfluid/keeper/epoch.go:36]: Ah I didn't see the log line in the method itself, seems fine as is, was using the previous call in hooks as the example

**Pull request merged**